apps/code-examples/src/app/code-examples/datetime/date-range-picker/custom-calculator/demo.component.ts

```diff
@@ -55,7 +55,7 @@ export class DemoComponent {
       shortDescription: 'Date before today',
       type: SkyDateRangeCalculatorType.Before,
       validate: (value): ValidationErrors | null => {
-        if (value && value.endDate && value.endDate > new Date()) {
+        if (value?.endDate && value.endDate > new Date()) {
```

Comment:
can this just be `value?.endDate > new Date()`? undefined won't be greater than a date, if the property doesn't exist

Reply:
It cannot (with this ruleset). The ruleset requires that values being compared as numbers are actually number values.